c2rust-analyze/src/borrowck/mod.rs

```diff
@@ -347,7 +364,7 @@ fn construct_adt_origins<'tcx>(
     ltcx.arena().alloc_from_iter(origins)
 }
 
-fn assign_origins<'tcx>(
+pub fn assign_origins<'tcx>(
```

Comment:
Does this need to be public?  It looks like the only new use of it in this PR is in `borrowck::type_check`, which can see it regardless.

Reply:
3303860a